.github/workflows/publish_helm_charts.yml

```diff
@@ -0,0 +1,42 @@
+# this is WIP and needs a revision and testing before merging. Also we need to add this secrets in GH secrets
+name: Publish helm charts
+on:
+# Will only run when release is published. 
+  push:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  publish-helm-charts:
+    runs-on: ubuntu-20.04
+    container: hypertrace/helm-gcs-packager:0.3.0
+    # GITHUB_REF will be refs/tags/docker-MAJOR.MINOR.PATCH
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}       
+   
+      - name: package and release charts
+        env:
+          HELM_GCS_CREDENTIALS: ${{ secrets.HELM_GCS_CREDENTIALS }}
+          HELM_GCS_REPOSITORY: ${{ secrets.HELM_GCS_REPOSITORY }}
+        run: |
+            CHART_VERSION=$(echo ${GITHUB_REF} | cut -d/ -f 3)
```

Comment:
Please add a comment on what you are trying to achieve here, e.g. `https://github.com/my/repo` -> `xxxx` also that would make it explicit on what GITHUB_REF look like (maybe it is ssh)?

Reply:
I see, it is noting related to repo. That is why we need such comments, like in https://github.com/hypertrace/attribute-service/pull/43/files#diff-eeb8c8da76f393166c42141fcde9447d8df7ddce3414ae8dbe533c90565c13c0R19 :D

Reply:
Done